utils/sysfs/sysfs_x86.go

```diff
@@ -0,0 +1,19 @@
+// +build x86
+
+// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sysfs
+
+var isX86 = true
```

Comment:
nit: sorry to go back and forth on this, but maybe as another idea instead of the `isX86` package variable, consider passing `arch` as an argument in `IsCPUOnline`. That way in the tests you can easily just pass `amd64` or `arm64` and in the calling code of `IsCPUOnline` use `runtime.GOARCH` during runtime. Totally up to you if you want leave it as is or do that approach, just wanted to bring it up as another option.

Reply:
At the moment I prefer to leave it as it is.